src/components/Modal/index.tsx

```diff
@@ -8,9 +8,16 @@ type ModalProps = {
   children: ReactNode | ReactNodeArray;
   isOpen: boolean;
   closeModal: () => void;
+  shouldCloseOnOverlayClick?: boolean;
```

Comment:
Is this something we would want to allow as an override, or should we always have the modal close on overlay click?